### ISSUE TITLE
Change urlpatterns to match Django1.10

### ISSUE THREAD

**Issue by noamgal** [concentricsky/django-tastypie-swagger#117]
django.conf.urls.patterns() will be removed in Django1.10.
https://docs.djangoproject.com/en/dev/internals/deprecation/#deprecation-removed-in-1-10

There are 2 pull requests to fix that:
https://github.com/concentricsky/django-tastypie-swagger/pull/116 (mine , very defensively for django version)
and https://github.com/concentricsky/django-tastypie-swagger/pull/115

@iankpconcentricsky - please approve one of them

Thanks

**Comment by DarthLabonne** [comment]
+1

**Comment by DarthLabonne** [comment]
About your patch, it should be:

``` python
main_ver = int(django_version[0])
secondary_ver = int(django_version[1])
```

else, you get:

```
TypeError: int() argument must be a string or a number, not 'list'
```

**Comment by Tsangares** [comment]
@iankpconcentricsky I second that pull request

**Comment by noamgal** [comment]
@iankpconcentricsky @Tsangares so, what is the status of this? isn't that urgent for django 1.10?

**Comment by Tsangares** [comment]
@noamgal Yes, it looks like there is another pull request #118. I could not get this django-tastypie-swagger to work sadly. Using pull #118, and #116, did not completely solve my issues. I do admit, I did not try very hard. But I was not eager to go debug someone else's code, I just started writing my own documentation because my api is small.

**Comment by joshkehn** [comment]
@noamgal Will try and find time to review this PR this week.

**Comment by joshkehn** [comment]
Linking #118 that covers this as well.

**Comment by andrijan** [comment]
Any news on this?

**Comment by pg1671** [comment]
+1

**Comment by AlSayedGamal** [comment]
PR #121  should solve this
